src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -56,6 +59,29 @@ public class IPAddress
         //   1: Terminating null byte
         internal const int NumberOfLabels = IPAddressParserStatics.IPv6AddressBytes / 2;
 
+        // When IPv6 support was added to the .NET Framework, the public Address property was marked as Obsolete.
+        // The public obsolete Address property has not been carried forward in .NET Core, but remains here as
+        // internal so internal types that understand IPv4 can still access it without obsolete warnings.
+        internal long Address
+        {
+            get { return _addressOrScopeId; }
```

Comment:
Should we assert InternalIsIPv4?  Or are there cases where Address is accessed when IPv6?